docs/admin/memory.rst

```diff
@@ -74,30 +73,64 @@ Managing translation memory
 User interface
 ++++++++++++++
 
+Available by clicking ones user avatar in the top-right corner of the
+interface and selecting "Translation memory" from the dropdown menu.
+Entries attributed to the user are listed as a total, and for each respective
+project contributed to, with options to download (as JSON, or TMX) or delete them.
+
 .. versionadded:: 3.2
 
-In the basic user interface you can manage per user and per project translation
+In the basic user-interface you can manage per-user and per-project translation
 memories. It can be used to download, wipe or import translation memory.
 
+.. versionadded:: 3.8
+
+There are multiple options for downloading the translation memory of the whole platform.
+
 .. hint::
 
-    Translation memory in JSON can be imported into Weblate, TMX is provided for
-    interoperability with other tools.
+    Translation memory in JSON can be imported into Weblate, the TMX format
+    is provided for interoperability with other tools.
 
 .. seealso::
 
     :ref:`schema-memory`
 
 .. image:: /screenshots/memory.png
 
+.. versionadded:: 3.8
+
+You can search for translations in the view built for this.
+
+Admin interface
++++++++++++++++
+
+.. versionadded:: 3.5
+
+There is a platform-wide interface to manage the translation memory.
+
+.. versionadded:: 4.12
+
+It has the same options as the user-interface, but also allows
+rebuilding parts of or the entire translation memory.
+All old entries can be flushed and re-created from a component or project by
+selecting "Administration" from amidst the different tabs
+at the top of the screen, and then "Translation memory".
+
 Management interface
 ++++++++++++++++++++
 
-There are several management commands to manipulate the translation memory
-content. These operate on the translation memory as whole, unfiltered by scopes (unless
-requested by parameters):
+Several management commands can manipulate translation-memory content.
+These operate on the translation memory as a whole, unfiltered by scopes
+(unless requested by parameters):
 
 :djadmin:`dump_memory`
     Exports the memory into JSON
 :djadmin:`import_memory`
     Imports TMX or JSON files into the translation memory
+
+.. versionadded:: 4.14
+
+The Weblate API covers the translation memory.
+This allows automating manipulation for different purposes,
+or based on events in the translation cycle.
```

Comment:
This part is guesswork that could possibly be improved by some realistic scenarios.